docker-compose-generator/docker-fragments/bitcoin-eclair.yml

```diff
@@ -0,0 +1,56 @@
+version: "3"
+
+services:
+  bitcoind:
+    environment:
+      BITCOIN_NETWORK: ${NBITCOIN_NETWORK:-regtest}
+      BITCOIN_EXTRA_ARGS: |
+        zmqpubrawblock=tcp://0.0.0.0:28332
+        zmqpubrawtx=tcp://0.0.0.0:28333
+        deprecatedrpc=signrawtransaction
+        addresstype=p2sh-segwit
+        rpcuser=ceiwHEbqWI83
+        rpcpassword=DwubwWsoo3
+  eclair_bitcoin:
+    restart: unless-stopped
+    stop_signal: SIGINT
+    image: acinq/eclair:v0.3
+    environment:
+      PUBLIC_HOST: ${BTCPAY_HOST}
+      JAVA_OPTS: |
+        -Xmx256m
+        -Declair.printToConsole
+        -Declair.headless
+        -Declair.chain=${NBITCOIN_NETWORK:-regtest}
+        -Declair.server.binding-ip=0.0.0.0
+        -Declair.server.public-ips.0=${BTCPAY_HOST}
+        -Declair.server.port=9735
+        -Declair.api.enabled=true
+        -Declair.api.binding-ip=0.0.0.0
+        -Declair.api.port=8080
+        -Declair.node-alias=${LIGHTNING_ALIAS}
+        -Declair.api.password=DwubwWsoo3
+        -Declair.bitcoind.host=bitcoind
+        -Declair.bitcoind.rpcport=8080
+        -Declair.bitcoind.rpcuser=ceiwHEbqWI83
+        -Declair.bitcoind.rpcpassword=DwubwWsoo3
+        -Declair.bitcoind.zmqblock=tcp://bitcoind:28332
+        -Declair.bitcoind.zmqtx=tcp://bitcoind:28333
+    expose:
+      - "9735" # server port
+      - "8080" # api port
+    volumes:
+      - "bitcoind_dir:/etc/bitcoin"
+      - "eclair_bitcoin_datadir:/data"
+    links:
+      - bitcoind
+  btcpayserver:
+    environment:
+      BTCPAY_BTCLIGHTNING: "type=eclair;server=http://eclair_bitcoin:8080;password=DwubwWsoo3;bitcoin-host=bitcoind:43782;bitcoin-auth=ceiwHEbqWI83:DwubwWsoo3"
+    volumes:
+      - "eclair_bitcoin_datadir:/etc/eclair_bitcoin"
+    links:
+      - eclair_bitcoin
+      - bitcoind
```

Comment:
@NicolasDorier  Should I remove the direct dependency to bitcoind? We do not use the `GetDepositAddress` in btcpay right now and  would not need it if @araspitzu  implements https://github.com/ACINQ/eclair/issues/1006

Reply:
Yes, should remove this.